content/community/twitch.md

```diff
@@ -0,0 +1,10 @@
++++
+title = "Twitch"
+# https://en.wikipedia.org/wiki/Twitch_(service)
+description = "Twitch is an online live streaming platform."
+date = "2022-07-23T03:55:42-04:00"
+draft = false
+logo = "/images/3rd-party/Twitch.svg"
```

Comment:
The filename for the logo is `TwitchLogo.svg`

![image](https://user-images.githubusercontent.com/10230166/219963111-dcd946d7-0cbf-4118-9571-625d81509f10.png)

```suggestion
logo = "/images/3rd-party/TwitchLogo.svg"
```